client/coral-embed-stream/src/containers/Embed.js

```diff
@@ -84,6 +85,9 @@ export const withQuery = graphql(EMBED_QUERY, {
       hasComment: commentId !== '',
       excludeIgnored: Boolean(auth && auth.user && auth.user.id),
     },
+    reducer: (previousResult, action, variables) => {
+      return reduceEditCommentActionsToUpdateStreamQuery(previousResult, action, variables);
```

Comment:
Doing it this way is what felt most write after reading http://dev.apollodata.com/react/cache-updates.html .

it says

> We recommend using the reducer option, except when there’s a good reason to use updateQueries instead (eg. if it would make your app much easier to understand and maintain).